pkg/kubelet/kubelet.go

```diff
@@ -1623,6 +1629,13 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 				return
 			}
 			kl.podManager.UpdatePods(u, podSyncTypes)
+			if u.Op == cache.ADD {
+				// TODO: Break out SyncPods and check admission etc
+				for _, p := range u.Pods {
+					kl.podWorkers.UpdatePod(p, nil, func() {})
+				}
+				continue
+			}
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
```

Comment:
I suggest we use time.Tick here, which can guarantee even the update contains just creation, we will still sync every `resyncInterval`:
http://play.golang.org/p/LGO3z5for0